.github/workflows/build.yml

```
@@ -124,11 +127,20 @@ jobs:
           PACKAGE_URL: honnef.co/go/tools/cmd/staticcheck
           PACKAGE_VERSION: ${{ steps.setup-env.outputs.staticcheck-version }}
         run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
-      - name: Install Terraform-docs
-        env:
-          PACKAGE_URL: github.com/terraform-docs/terraform-docs
-          PACKAGE_VERSION: ${{ steps.setup-env.outputs.terraform-docs-version }}
-        run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
+      # We are temporarily using @mcdonnnj's forked branch of terraform-docs
+      # until his PR: https://github.com/terraform-docs/terraform-docs/pull/745
+      # is approved. This temporary fix will allow for ATX Header Support when
+      # terraform-docs is ran during lint.
+      - name: Clone ATX Headers branch from Terraform-docs Fork
+        run: |
+          git clone --branch $BRANCH_NAME --single-branch \
+           --depth=$DEPTH $REPO_URL /tmp/terraform-docs
```

In line with the previous suggestion.

```suggestion
          git clone --single-branch \
           --branch $TERRAFORM_DOCS_BRANCH_NAME \
           --depth=$TERRAFORM_DOCS_REPO_DEPTH \
           $TERRAFORM_DOCS_REPO_URL /tmp/terraform-docs
```